models/article_id.model.js

```diff
@@ -14,8 +8,8 @@ const fetchArticle = (id) => {
         return Promise.reject({ status: 404, msg: "Article not found" });
       }
 
-      return article[0];
```

Comment:
I see now why you added those square brackets higher up. I'd rather you return this as article[0] for readability please! I will approve the pr for now but please address this before you submit your next one!

Reply:
so get rid of other brackets and add the [0] in the model, because I shouldnt be using [] in the controller, got it

Reply:
Thank You For My Feedback @Rosem10 <3